includes/Modules/Analytics_4/GoogleAnalyticsAdmin/Proxy_GoogleAnalyticsAdminProvisionAccountTicketRequest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class Proxy_GoogleAnalyticsAdminProvisionAccountTicketRequest
+ *
+ * @package   Google\Site_Kit\Modules\Analytics_4\GoogleAnalyticsAdmin
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics_4\GoogleAnalyticsAdmin;
+
+use Google\Site_Kit_Dependencies\Google\Service\GoogleAnalyticsAdmin\GoogleAnalyticsAdminV1betaProvisionAccountTicketRequest;
+
+/**
+ * Class for representing a proxied account ticket provisioning request body.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Proxy_GoogleAnalyticsAdminProvisionAccountTicketRequest extends GoogleAnalyticsAdminV1betaProvisionAccountTicketRequest {
+	/**
+	 * The site ID.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	public $site_id = '';
+
+	/**
+	 * The site secret.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	public $site_secret = '';
+
```

Comment:
When running the code, following the instructions in the QAB, I found the call to `create-account-ticket` returns an `invalid 'redirect_uri'` error. I tried this both locally and on a new InstaWP site.

Digging into this I found the `redirectUri` parameter is sent in camel-case, where it should be snake-case:

![image](https://user-images.githubusercontent.com/18395600/226676527-f0fc1375-a9e2-4cab-9122-8e7dd722c15e.png)

It looks like this is a result of the camel-case property in [`GoogleAnalyticsAdminV1betaProvisionAccountTicketRequest` ](https://github.com/googleapis/google-api-php-client-services/blob/72e0eacbd51131c954da05edf110d9774f0f5af0/src/GoogleAnalyticsAdmin/GoogleAnalyticsAdminV1betaProvisionAccountTicketRequest.php#L27) being reflected to create the request.

Adding a snake-case public `$redirect_uri` property here and overriding the setter `setRedirectUri()` to populate it fixed the issue for me. I think we should also override `getRedirectUri()` for consistency.

```suggestion

	/**
	 * The redirect URL.
	 *
	 * @since n.e.x.t
	 * @var string
	 */
	public $redirect_uri = '';

	/**
	 * Sets the redirect URL.
	 *
	 * @since n.e.x.t
	 *
	 * @param string $redirect_uri The redirect URL.
	 */
	public function setRedirectUri( $redirect_uri ) {
		$this->redirect_uri = $redirect_uri;
	}

	/**
	 * Gets the redirect URL.
	 *
	 * @since n.e.x.t
	 */
	public function getRedirectUri() {
		return $this->redirect_uri;
	}
```

Reply:
Thanks, I ran into this in my testing yesterday as well, but it's been fixed on the service side.

We shouldn't override parameters of the base request class, this is essentially just for including the site_id and secret 👍